src/Converter/Feature2Mesh.js

```diff
@@ -411,7 +404,7 @@ function featureToMesh(feature, options) {
             mesh = featureToLine(feature, options);
             break;
         case FEATURE_TYPES.POLYGON:
-            if (options.extrude) {
+            if (feature.style.fill.extrusion_height) {
```

Comment:
Could you change the doc for the `featureToMesh` method ?  It still mentions `options.altitude`, `options.extrude` and `options.color`.

Reply:
Thanks !